.gitmodules

```diff
@@ -45,3 +45,8 @@
 [submodule "lib/fsp"]
 	path = lib/fsp
 	url = https://github.com/renesas/fsp.git
+[submodule "lib/wiznet"]
+	path = lib/wiznet5k
+	url = https://github.com/andrewleech/wiznet_ioLibrary_Driver.git
+	# Requires https://github.com/Wiznet/ioLibrary_Driver/pull/120
+	# url = https://github.com/Wiznet/ioLibrary_Driver.git
```

Comment:
Is your PR merged upstream?

Also, please undo the change to stm32lib.

Reply:
My PR there hasn't had any feedback no. I'm happy to turn this into a "copy into micropython" or move this to a https://github.com/micropython/wiznet_ioLibrary_Driver.git repo?

Sorry I missed that stm32lib change, it must have snuck in during the most recent rebase onto master. Will fix.